webapp/app/models/session.js

```diff
@@ -1,6 +1,10 @@
 import DS from 'ember-data';
 
 export default DS.Model.extend({
+  nId: function() {
+    return +this.get('id');
```

Comment:
id (ember internal) is a string. When trying to sort by it you get a text sort and not a number sort. This is atrick I saw on multiple implementations. Will changing the name to integerID helps?